cmd/gobgpd/main.go

```diff
@@ -88,7 +91,14 @@ func main() {
 		runtime.GOMAXPROCS(opts.CPUs)
 	}
 
+	metricRegistry := prometheus.NewRegistry()
+
 	if !opts.PProfDisable {
```

Comment:
Should this line be changed? Otherwise if you disable pprof, you will also disable metrics, right?